plugins/connection/podman.py

```diff
@@ -82,10 +84,15 @@ class Connection(ConnectionBase):
     # users will start containers and fail to connect to them.
     has_pipelining = False
 
+    def _get_container_id(self):
+        _container_id = self.get_option('remote_addr')
+        if _container_id is None and self._play_context.remote_addr is not None:
+            _container_id = self._play_context.remote_addr
+        return _container_id
```

Comment:
please remove it if you don't use this function

Reply:
Didn't you want to remove this function and `_container_id` completely?

Reply:
That was the suggestion above, but if I do this there is literally no code changes in the PR (so I suspects it won't fix the problem). 

More importantly, the tests failed complaining about the missing _container_id. So I put both back, to see if the tests passes this time.

Reply:
If I understood @adrianlzt correctly, you don't need to change a code, but just to add these lines:
```
+          - name: inventory_hostname
+          - name: ansible_podman_host
```